server/router/driver.ts

```diff
@@ -20,11 +64,6 @@ router.get('/:id', validateUser('User'), (req, res) => {
   db.getById(res, Driver, id, tableName);
 });
 
-// Get all drivers
-router.get('/', validateUser('Admin'), (req, res) => {
-  db.getAll(res, Driver, tableName);
-});
-
```

Comment:
why was this removed? we still use this endpoint on the frontend

Reply:
Actually it's not removed. Now it's the first endpoint in driver.ts.